sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/client_options.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the based options to create a Key Vault client.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/internal/client_options.hpp>
+#include <azure/core/response.hpp>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Common {
+
+  /**
+   * @brief Available and supported service versions.
+   *
+   */
+  enum class ServiceVersion
+  {
+    V7_0,
+    V7_1,
+    V7_2
+  };
```

Comment:
Each version enum value needs docs.

I would like to drill a bit deeper into how we are doing multi-version support.